docker-for-mac/edge-release-notes.md

```diff
@@ -11,6 +11,21 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.3.3.2
+2020-07-date
+
+Download
+
+### Upgrades
+
+- [Docker ECS integration v1.0.0-beta.2](https://github.com/docker/ecs-plugin/releases/tag/v1.0.0-beta.2){: target="_blank" class="_”}
+- [Docker ACI integration 0.1.10](https://github.com/docker/aci-integration-beta/releases/tag/v0.1.10){: target="_blank" class="_”}
+
+### Bug fixes and minor changes
+
+- Mutagen uses the `.dockerignore` file when creating a session to filter the list of synchronized files. See [docker/for-mac#4595](https://github.com/docker/for-mac/issues/4595#issuecomment-634605312).
```

Comment:
I don't think this is the right link. It doesn't seem to mention .dockerignore. In fact, the whole ticket doesn't seem to mention .dockerignore. I'm not sure what the best link is though. @djs55 Where was it first discussed?

Reply:
https://github.com/docker/for-mac/issues/4621#issuecomment-658109369 seems the correct issue.

Reply:
Will update this in the Desktop changelog

Reply:
I think we can just link to the whole ticket (4621), rather than a comment within it.

Reply:
Fixed